webbpsf/webbpsf_core.py

```diff
@@ -1745,6 +1745,17 @@ def load_wss_opd_by_date(self, date=None, choice='closest', verbose=True, plot=F
         opd_fn = webbpsf.mast_wss.get_opd_at_time(date, verbose=verbose, choice=choice, **kwargs)
         self.load_wss_opd(opd_fn, verbose=verbose, plot=plot, **kwargs)
 
+    def _label_wl (nwavelengths, wavelength_slices):
```

Comment:
If you're defining this as class method, it needs a `self` at the start. But since this doesn't depend on `self`, it might be better placed as a standalone function in utils rather than as part of the class definition.

Reply:
yup, realized that right away, have it fixed locally but will transfer it to utils.